### ISSUE TITLE
Unable to downgrade from (Emu)11.0.0 to 2.1.0 through PlaiSysUpdater in RedNAND

### ISSUE THREAD

**Issue by theTarkus** [Plailect/Guide#196]
I've retried from scratch twice now and I've the same error both times. On top of that, I have also restored the emuNand backup multiple times and still to no avail.
(https://github.com/Plailect/Guide/wiki/Part-4-(2.1.0-Downgrade) Downgrading, Step 11)

Inside of the RedNAND (Version (Emu)11.0.0), when running PlaiSysUpdater I receive an error, which when searched, I've only been able to find one other person with this issue. (https://www.reddit.com/r/3dshacks/comments/4gfo4k/3dshacking_qa_general_too_many_hax_edition/d2iwvjq)

**The output with error:**
```
HAX INIT...
HAX SUCCESS

Init -> GOOD
Check device -> o3DS USA
Check update info -> o3DS USA

Check update integrity...

Can't get cia information (hax didn't succeed?)  # <-- Error in red
press (A) to exit...
```

**What I've tried:**
* Starting all the way from scratch, remaking the RedNAND.
* Deleting and re-transferring "/updates/" directory (from 2.1.0) to the root directory of the 3DS SD card.
* Restoring the RedNAND backup. (Many times)
* Putting the firmware.bin from "Firmware_NTR_o3DS" into /luma/ directory; (ala https://gbatemp.net/threads/cant-downgrade-from-9-2-to-2-1-0.430263/#post-6431619)

I've run out of ideas other than trying to downgrade from (Emu)9.2.0 itself.
Any ideas?

**Comment by nickthegamer5** [comment]
Have you tried to redownload the 2.1 files? They be corrupt.

**Comment by theTarkus** [comment]
Thank you so much, I redownloaded it from Mega on Ubuntu. Downloading it on Windows from the Google Drive mirror caused me problems, I'm not sure if it was Windows or the Google Drive Mirror is corrupt.

I verified the SHA256 hash after downloading it and all was good.

Putting the firmware.bin from "Firmware_NTR_o3DS" into /luma/ directory, may or may not have helped the install. Since I tried it previously without the file and it failed, then I added the file and it seems to have worked now.
